tests/python/relay/test_autotvm_task_extraction.py

```diff
@@ -121,7 +121,21 @@ def get_net(batch, in_dim, out_dim, dtype, out_dtype):
     tasks = autotvm.task.extract_from_program(mod, target=target, params=params, ops=(dense,))
     assert len(tasks) == 1 and tasks[0].name == "dense_int8.cuda"
 
+def test_task_extraction_for_dense_int8_vnni():
```

Comment:
Do we need to check the ISA here?

Reply:
I am not sure if AlterOPLayout would work correctly when the ISA is absent but the target is specified. If yes, ISA check is not needed here.

Reply:
@yangulei Double checked _alter_dense_layout() in /tvm/python/tvm/topi/x86/dense_alter_op.py. The AlterOPLayout should work if the ISA is missing.

Reply:
OK.
You should add one extra blank line before and after this newly added function to pass the CI lint.